src/common/camera_control.c

```diff
@@ -20,6 +20,7 @@
 #endif
 #include "common/camera_control.h"
 #include "common/exif.h"
+#include "common/datetime.h"
```

Comment:
Probably superfluous since nothing else if changed here. I'll test and remove if possible.

Reply:
And finally this is needed.